app/components/elements/forms/checkbox_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="form-check mb-4">
   <%= render Elements::Forms::LabelComponent.new(form:, field_name:, label_text: label, hidden_label:, default_label_class: 'form-check-label') %>
   <%= render Elements::Forms::HelpTextComponent.new(id: help_text_id, help_text:) %>
-  <%= form.checkbox field_name, class: 'form-check-input', required:, aria: field_aria, data: %>
+  <%= form.checkbox field_name, class: 'form-control form-check-input', required:, aria: field_aria, data:, style: styles %>
```

Comment:
Using both form-control and form-check-input doesn't seem like correct BS.

Reply:
Yeah, I was thinking about this and realized it was a bit of an anti-pattern and that I should be setting the border on the `form-check-input` instead of using `form-control` to get the border but then have to override padding. Fixed.